website/blog/2023-12-06-release-1.6.md

```diff
@@ -0,0 +1,250 @@
+- **Enhancement to show environment colums on Containers and Pods lists**: 
+- **Extension API improvements**: Another set of improvements to the extension API enabling more goodness from 🦭 Podman Desktop's extensions
+
+Podman Desktop 1.6 is now available. [Click here to download it](/downloads)!
+
+---
+
+## Release Details
+
+### Minikube featured extension
+For developers who need to run Kubernetes locally and reproduce an environment close to production for development and experimentation purposes, Podman Desktop allows users to easily set up that environment on a local machine. There are two distincts extensions providing the capability to configure a Kubernetes cluster locally, you can either choose between [Kind](https://kind.sigs.k8s.io/) or [minikube](https://minikube.sigs.k8s.io/docs/).
+
+The minikube extension allows to setup a Kubernetes cluster locally running in a container! Yeah, you read well, in a container, in a similar fashion than Kind. The advantage is that it's lighter and faster to start. With minikube, one of the advantage, is that you can build your images locally with podman and get them automatically available in your local Kubernetes cluster -  which will speed up your turnarounds when you want to test your application. If you want to learn more about the [following blog post](https://podman-desktop.io/blog/sharing-podman-images-with-kubernetes-cluster).
+
+![Minikube-feature-extension](img/podman-desktop-release-1.6/minikube-feature-extension.png)
+
+
+### Command-Line Tools Configuration: Compose and Kubectl
+Configuring and managing your setup is getting easier with the addition of a new section in the Settings to manage Command-line tools. In Podman Desktop, extensions can bring dependencies and tools that are going to be necessary for working properly. 
+
+Two Command-Line tools are getting registered at the moment Compose and kubectl:
+- Compose-cli is registered by the Compose extension. 
+- kubectl is a dependency needed with Minikube extension.
+
+![cli-tools](img/podman-desktop-release-1.6/cli-tools.png)
+
+From the settings you can see the command-line tools that are installed, you can see the version - and when a new version is available, you'll get a small notification to allow you easily updating that dependency. 
+
+![Compose-Update](img/podman-desktop-release-1.6/compose-update.png)
+
+
+### Kubernetes Contexts Manager
+We are introducing a new screen available from the settings allowing to easily manage your Kubernetes contexts. Podman Desktop was already providing the handy context switcher available from the status bar, but when you get to work with multiple Kubernetes environments, it's not uncommon to ended with a big and long list of Kubernetes contexts.
+
+The new Kubernetes Contexts screen allows you to easily see all your registered Kubernetes contexts. You can use that screen to cleanup your registered contexts, and also to define the context you want to be using by default.
```

Comment:
'You can use that screen to cleanup your registered contexts, and also to define the context you want to be using by default.' ->
'You can use the screen to clean up your registered contexts, or set the current (default) context.'